### PR TITLE
README.md update

### PR DESCRIPTION
**Pull request by Reictangle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,54 @@
+
 # DexScript - ALPHA
+<img src="https://i.imgur.com/uKfx0qO.png" width="200"> 
 
-DexScript is a set of commands developed by DotZZ on September 14, 2024, used for **BallsDex and CarFigures** Discord bot forks.
-Below is a guide on how to use DexScript, and what DexScript is capable of.
-Currently, DexScript is in alpha. DexScript will have more commands, the ability to support more models, and overall quality-of-life improvements as time passes.
+## What is DexScript?
+DexScript is a set of commands made by ``DotZZ``. The commands simplify things like editing, adding, and deleting things you want in your bot. Basically making your life easier.
```
Comment (Dotsian) [inline README.md:6]: I would recommend changing "things" to models

File: README.md [hunk 2]
```diff
@@ -1,14 +1,54 @@
...
+- ATTACK 
+- RARITY 
+- ENABLED 
+- TRADEABLE 
+- EMOJI_ID 
+- WILD_CARD 
+- COLLECTION_CARD 
+- IMAGE_CREDITS 
+- CAPACITY_NAME 
+- CAPACITY_DESCRIPTION
```
Comment (Dotsian) [inline README.md:34]: These fields are different depending on which BallsDex fork you use. If you use CarFigures, these fields won't work. I would recommend to delete this section as you can use the `LIST` command.

Comment (Reictangle) [reply README.md:34]: Fair enough.

File: README.md (hunk 1 above)
Comment (Reictangle) [reply README.md:6]: Will do.

**Pull request merged**